### PR TITLE
[0.26] - Bump

### PR DESCRIPTION
**Pull request by Quentinchampenois**
#### Description 

Bump code to v0.26

#### Tasks

- [x] Add webpacker
- [x] Update configs / secrets
- [x] Fix controller overrides
- [x] Remove / update views overloads
- [x] update specs
- [x] Check / fix font with webpacker

### CODE REVIEW

**Comment by Quentinchampenois** [comment]
Before merge we planned to update fonts in CSS

File: Gemfile [hunk 1]
```diff
@@ -2,30 +2,27 @@
...
 ruby RUBY_VERSION
 
 gem "decidim", git: "https://github.com/decidim/decidim.git", branch: DECIDIM_VERSION
 
-gem "decidim-decidim_awesome", "~> 0.7.0"
-gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer.git", branch: "master"
-gem "omniauth-publik", git: "https://github.com/OpenSourcePolitics/omniauth-publik", branch: "v0.0.9"
+# gem "decidim-decidim_awesome", "~> 0.7.0"
+# gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer.git", branch: "master"
+# gem "omniauth-publik", git: "https://github.com/OpenSourcePolitics/omniauth-publik", branch: "v0.0.9"
```
Comment (paulinebessoles) [inline Gemfile:13]: Could you uncomment those 3 gems and update the required branch? We still need them for the 0.26 version of this platform.

File: app/packs/stylesheets/decidim/_fontface.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
+@font-face {
```
Comment (paulinebessoles) [inline app/packs/stylesheets/decidim/_fontface.scss:1]: Could you fix the _fontface.scss file so it works with webpacker? You can have an example here: https://github.com/OpenSourcePolitics/decidim-reneweu/pull/9/files

File: app/views/decidim/devise/registrations/new.html.erb [hunk 3]
```diff
@@ -45,7 +45,7 @@
             </div>
 
             <div class="field">
-              <%= f.password_field :password, help_text: t(".password_help", minimun_characters: NOBSPW.configuration.min_password_length), autocomplete: "off" %>
+              <%= f.password_field :password, help_text: t(".password_help", minimun_characters: ::PasswordValidator::MINIMUM_LENGTH), autocomplete: "off" %>
```
Comment (paulinebessoles) [inline app/views/decidim/devise/registrations/new.html.erb:48]: Why did you update this line? It seems it creates a problem with the sign up form.
`uninitialized constant NOBSPW`

File: app/views/layouts/decidim/_wrapper.html.erb [hunk 4]
```diff
@@ -10,7 +10,7 @@ if respond_to?(:current_component) && current_component && can_be_managed?(curre
 end
 %>
 
-<div class="off-canvas-wrapper" id="app">
+<div class="off-canvas-wrapper">
```
Comment (paulinebessoles) [inline app/views/layouts/decidim/_wrapper.html.erb:13]: I think this was a security request from the audit

**Review by paulinebessoles — CHANGES_REQUESTED** [review]

**Comment by armandfardeau** [comment]
Hi this ready? I've changed the status to draft, feel free to undraft when over.

File: app/views/decidim/devise/registrations/new.html.erb (hunk 3 above)
Comment (eliegaboriau) [reply app/views/decidim/devise/registrations/new.html.erb:48]: It seems ok to me and to the CI

**Review by paulinebessoles — APPROVED** [review]
Sorry, my mistake, fonts are working well!

**Review by armandfardeau — APPROVED** [review]

**Pull request merged**